main.tf

```diff
@@ -3,29 +3,21 @@ This file is used to implement the ROOT module.
 *********************************************************************/
 
 resource "ibm_db2" "db2" {
-  name              = var.db2_instance_name
-  resource_group_id = var.resource_group_id
-  location          = var.region
-  plan              = var.subscription_id == null ? "performance" : "PerformanceSubscription"
-  service           = "dashdb-for-transactions"
-  high_availability = var.enable_high_availability ? "yes" : "no"
-  service_endpoints = var.service_endpoints
-
-  parameters = {
-    "oracle_compatibility" : var.enable_oracle_compatibility ? "yes" : "no",
-    "version" : "12",
-    "node_type" : var.node_type,
-    "subscription_id" : var.subscription_id == null ? "" : var.subscription_id
-  }
+  name                 = var.db2_instance_name
+  service              = "dashdb-for-transactions"
+  plan                 = var.subscription_id == null ? "performance" : "PerformanceSubscription"
+  location             = var.region
+  resource_group_id    = var.resource_group_id
+  service_endpoints    = var.service_endpoints
+  instance_type        = var.node_type == "bx2.1x4" ? null : var.node_type
+  high_availability    = var.enable_high_availability ? "yes" : "no"
+  oracle_compatibility = var.enable_oracle_compatibility ? "yes" : "no"
+  subscription_id      = var.subscription_id
+  tags                 = var.tags
 
   timeouts {
     create = "720m"
     update = "60m"
     delete = "30m"
   }
 }
-
-resource "ibm_resource_tag" "tags" {
```

Comment:
this will be a breaking change anyway - ensure to include that in release note summary

Reply:
updated release note